packages/core/src/shader/GLProgram.ts

```diff
@@ -54,6 +71,7 @@ export class GLProgram
     {
         this.uniformData = null;
         this.uniformGroups = null;
+        this.uniformBufferBindings = null;
```

Comment:
Ignore this if the omission was intentional.

```suggestion
        this.uniformDirtyGroups = null;
        this.uniformBufferBindings = null;
```